core/templates/dev/head/editor/ExplorationEditor.js

```diff
@@ -172,6 +172,11 @@ oppia.controller('ExplorationEditor', [
     var _ID_TUTORIAL_PREVIEW_TAB = '#tutorialPreviewTab';
     var _ID_TUTORIAL_SAVE_BUTTON = '#tutorialSaveButton';
 
+    var screenWidth = windowDimensionsService.getWidth();
+    if (screenWidth <= 1280) {
+      _ID_TUTORIAL_PREVIEW_TAB = '#tutorialPreviewSmall';
```

Comment:
Yes, that sounds good.

I just noticed, though, that you might run into a problem with the tutorial -- AFAICT the ng-joyride library uses a constant config, so you can't generate the tutorial config (and therefore the id to use) on the fly.

A possible way around this is to add an element of type 'function' to the tutorial config, just before the 'preview' card. Then, if the screen is small, that function opens the dropdown menu before proceeding to the 'preview' card. See if that makes it possible for you to use just one id in both the small- and large-width cases.